core/controllers/topics_and_skills_dashboard.py

```diff
@@ -98,13 +98,15 @@ def get(self) -> None:
                             self.user, topic_rights)
                     )
 
-        all_classrooms_dict = config_domain.CLASSROOM_PAGES_DATA.value
+        classrooms = classroom_config_services.get_all_classrooms()
+        all_classrooms_dict = [classroom.to_dict() for classroom in classrooms]
```

Comment:
Let's avoid creating dicts, can we just use the Classroom objects directly here? (See above.)